coordinator/src/main/scala/com/socrata/datacoordinator/service/collocation/CollocationLock.scala

```diff
@@ -0,0 +1,37 @@
+package com.socrata.datacoordinator.service.collocation
+
+import java.util.concurrent.TimeUnit
+
+import org.apache.curator.framework.CuratorFramework
+import org.apache.curator.framework.recipes.locks.InterProcessMutex
+
+sealed abstract class CollocationLockException(message: String, cause: Exception) extends Exception(message, cause)
+
+case class CollocationLockTimeout(millis: Long)
```

Comment:
Does it add any value to wrap the boolean return in a whole case class / option / etc. when there is really no information other than it is acquired or not to be had?